btk/draw_blends.py

```diff
@@ -437,54 +441,27 @@ def draw_single(self, cat, shift):
         Returns:
             gal (galsim.InterpolatedImage): The galsim profile of a single galaxy
         """
-
-        k = np.int(np.random.randn(1) * cat.size)
-        gal = cat.makeGalaxy(
-            k, gal_type="real", noise_pad_size=self.stamp_size * self.pixel_scale
-        )
+        k = int(np.random.rand(1)*len(self.cat))#catalog_line["btk_index"][0]
```

Comment:
I believe the galaxy id should be decided in the sampling function (because that is where we already have taken some entries in the catalog, it does not seem right to take an entirely different profile, plus the parameters like magnitude are bound to be wrong). So I guess the commented code should be the right one.